### PR TITLE
Custom tag management

### PR DESCRIPTION
**Pull request by Half-Shot**
Styling for https://github.com/matrix-org/matrix-react-sdk/pull/891

### CODE REVIEW

File: src/skins/vector/css/themes/_base.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@ $focus-bg-color: #dddddd;
 // button UI (white-on-green in light skin)
 $accent-fg-color: #ffffff;
 $accent-color: #76CFA6;
+$accent-color-disabled: #abd5c2;
```
Comment (t3chguy) [inline src/skins/vector/css/themes/_base.scss]: casing of hex characters :P

File: src/skins/vector/css/themes/_dark.scss [hunk 2]
```diff
@@ -12,6 +12,7 @@ $light-fg-color: #747474;
 // button UI (white-on-green in light skin)
 $accent-fg-color: $primary-bg-color;
 $accent-color: #76CFA6;
+$accent-color-disabled: #abd5c2;
```
Comment (t3chguy) [inline src/skins/vector/css/themes/_dark.scss]: same as above

**Comment by turt2live** [comment]
I'm super happy for this PR, and thanks to @t3chguy I have my hands on a build to play with.

I've found the following issues:
* Custom tags appear in the left sidebar below 'Historical', and do not show up in the tag management section
* Creating a new tag results in this:
  image
* Cannot drag rooms from 'Rooms' to anywhere (inter-tag dragging still works)
* Cannot delete tags (confirm button on dialog broken)
   ```
   RoomTags.js:127 Uncaught TypeError: Cannot read property 'protected' of undefined
       at Object.render (RoomTags.js:127)
       at d._renderValidatedComponentWithoutOwnerOrContext (ReactCompositeComponent.js:799)
       at d._renderValidatedComponent (ReactCompositeComponent.js:822)
       at d._updateRenderedComponent (ReactCompositeComponent.js:746)
       at d._performComponentUpdate (ReactCompositeComponent.js:724)
       at d.updateComponent (ReactCompositeComponent.js:645)
       at d.receiveComponent (ReactCompositeComponent.js:547)
       at Object.receiveComponent (ReactReconciler.js:125)
       at Object.updateChildren (ReactChildReconciler.js:109)
       at h._reconcilerUpdateChildren (ReactMultiChild.js:213)
   ```
* 'recent' ordering on a tag doesn't actually use 'recent' ordering (treated as 'manual')

**Comment by turt2live** [comment]
The other half of this PR was closed due to maintenance concerns - thanks again for the PR anyhow!

See https://github.com/matrix-org/matrix-react-sdk/pull/891#issuecomment-516803088 for details.

**Pull request closed without merging**